.travis.yml

```diff
@@ -5,16 +5,28 @@ go_import_path: github.com/pingcap/tidb
 go:
   - "1.11.3"
 
-env:
-    - TRAVIS_COVERAGE=0
-    - TRAVIS_COVERAGE=1
+cache:
+  directories:
+    - $GOPATH/pkg/mod
```

Comment:
We could cache the mod to speed up build. ([F.Y.R](https://docs.travis-ci.com/user/speeding-up-the-build/#caching-the-dependencies)).
(But I am not sure the disk space is enough, and I do not know what happened if we use it up).